drivers/spi/spi_ll_stm32.c

```diff
@@ -614,12 +614,6 @@ static int transceive(const struct device *dev,
 		return 0;
 	}
 
-#ifndef CONFIG_SPI_STM32_INTERRUPT
-	if (asynchronous) {
-		return -ENOTSUP;
-	}
-#endif
```

Comment:
Does it mean that async is supported even w/o dma?
(not clear from the commit message).

Reply:
yes, for SPI asynchronous transfer with or without DMA

Reply:
Ok, then maybe update the commit message, as this is not fully clear right now.

Reply:
Commit title not yet fully clear: "drivers: spi: stm32 spi driver supporting DMA in asynchronous mode"